lib/players/player.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Player do
+  defstruct symbol: nil, mode: nil, chooser: nil
+
+  def new(symbol, mode, chooser \\ :stdio) do
+    %Player{symbol: symbol, mode: mode, chooser: chooser}
+  end
+
+  def mark(board, selection, symbol) do
+    {status, result} = Board.mark(board, selection, symbol)
+  end
```

Comment:
en un juego de gato, es "responsabilidad" del player marcar el tablero? qué pasa si el espacio está ocupado? es responsabilidad del player decidir que vuelve a intentar?

Reply:
mmm aqui me confundo. antier te comente q yo le iba a pasar un board a mi player y mi player marcaria regresando un board... me dijiste q sonaba bien...

si el espacio esta ocupado, board le regresa un mensaje de error y se vuelve a intentar...

Reply:
Tio Bob: "The Single Responsibility Principle (SRP) states that each software module should have one and only one reason to change."